lib/msal-browser/docs/logout.md

```diff
@@ -159,22 +143,16 @@ const msal = new PublicClientApplication({
 })
 
 // Automatically on page load
-msal.logoutRedirect({
-    onRedirectNavigate: () => {
-        // Return false to stop navigation after local logout
-        return false;
-    }
-});
+msal.logoutRedirect();
 ```
 
-Now when a user logouts out of another application, your application's front-channel logout url will be loaded in a hidden iframe, and MSAL.js will clear its cache to complete single-sign out.
+Now when a user logouts out of another application, your application's front-channel logout url will be loaded, and MSAL.js will clear its cache to complete single-sign out.
```

Comment:
I thought front channel logout is still applicable in an iframe, when the storage partitioning is not enabled. 

Another note: May be we will benefit from adding reference to https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies or something similar.